pnpm-lock.yaml

```diff
@@ -26,8 +26,8 @@ devDependencies:
     specifier: ^4.1.10
     version: 4.1.10
   '@types/node':
-    specifier: ^20.11.6
-    version: 20.11.6
```

Comment:
@workers-frameworks @jculvey @dario-piotrowicz ... can you take a look at the types changes here to see if they make sense.